### ISSUE TITLE
KeyError: 'fuelRange'

### ISSUE THREAD

**Issue by slopemad** [itchannel/fordpass-ha#309]
Seeing these in home-assistant.log every 15 minutes (i.e. default poll interval), running 1.53-Beta4, in the UK, with an MY23 Mach E.

```
2023-10-11 09:41:43.758 ERROR (MainThread) [homeassistant.components.sensor] fordpass: Error on device update!
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 573, in _async_add_entity
    await entity.async_device_update(warning=False)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 936, in async_device_update
    await self.async_update()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 511, in async_update
    await self.coordinator.async_request_refresh()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 242, in async_request_refresh
    await self._debounced_refresh.async_call()
  File "/usr/src/homeassistant/homeassistant/helpers/debounce.py", line 93, in async_call
    await task
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 269, in async_refresh
    await self._async_refresh(log_failures=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 392, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 173, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 473, in _handle_coordinator_update
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 730, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 830, in _async_write_ha_state
    state, attr = self._async_generate_attributes()
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 774, in _async_generate_attributes
    attr.update(self.extra_state_attributes or {})
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/fordpass/sensor.py", line 406, in extra_state_attributes
    return self.get_value("attribute")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/fordpass/sensor.py", line 216, in get_value
    if self.data["fuelRange"] is None:
       ~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'fuelRange'
```

**Comment by itchannel** [comment]
Thanks, should be fixed in Beta 5 in 5 mins :)

**Comment by itchannel** [comment]
Fixed in 1.53 Beta 5
